bin/git-authors

```diff
@@ -43,12 +23,47 @@ authors() {
   fi
 }
 
-#
-# authors.
-#
+if [[ -z "$file" ]]; then
+  file=$(find . -mindepth 1 -maxdepth 1 \( -iname '*authors*' -o -iname '*contributors*' \) | head -n1)
+  if [[ -z "$file" ]]; then
+    file="AUTHORS"
+  fi
+fi
+
+if [[ $# = 0 ]]; then
+  authors >> "$file"
+  exit 0
+fi
+
+while [[ $# -gt 0 ]]; do
```

Comment:
Moving big sections around like the argument parsing makes it more difficult to read the diff. Can the argument parsing be moved back to where it was - at the top of the file?